README.md

```diff
@@ -1 +1,21 @@
 ### ReadMe Test
+def mod_func(a,b):
+  return a%b
+
+def addition_func(a,b):
+  return a+b
+
+def mult_func(a,b):
+  c=a*b
+  return c
+
+
+a=51
+b=11
```

Comment:
This line assigns the value 11 to the variable 'b'. No issues or improvements needed.